.gitignore

```diff
@@ -144,3 +144,4 @@ node_modules/
 /playground/**/.gitignore
 /playground/**/azure.yaml
 /playground/**/next-steps.md
+*.svclog
```

Comment:
Do we know where this file comes from?

Reply:
These are generated by the new VS IntPre, I think.

Reply:
![image](https://github.com/dotnet/aspire/assets/4403806/a407e5f1-174b-4f66-98c6-41c5c0b4a7f5)